randovania/games/samus_returns/gui/ui_files/msr_cosmetic_patches_dialog.ui

```diff
@@ -46,10 +46,17 @@
         <x>0</x>
         <y>0</y>
         <width>504</width>
-        <height>392</height>
+        <height>414</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
+       <item>
+        <widget class="QCheckBox" name="enable_remote_lua">
+         <property name="text">
+          <string>Enable automatic item tracker</string>
```

Comment:
Maybe "Enable automatic item tracker for solo worlds"? Would be nice to also have a label indicating that this forces exporting for Citra.

Reply:
It doesn't just work for solo worlds so saying that isn't correct

Reply:
Now I feel kinda dumb about this. I can go back and change this to explicitly mention solo worlds.